src/Actions/Pending.ts

```diff
@@ -15,13 +21,18 @@ const _hasPendingParity = async (conf: any, txRequest: any) => {
         params: [],
         id: 0o7
       },
-      (err: Error, res: any) => {
+      async (err: Error, res: any) => {
         if (err) reject(err);
 
         const hasTx =
           res &&
           res.result &&
-          !!res.result.filter((tx: any) => tx.to === txRequest.address).length;
+          !!await res.result.filter(async (tx: any) => {
```

Comment:
While testing with CLI I discovered that this block returns `True` when it should return `False`